csympy/lib/csympy_wrapper.pyx.in

```diff
@@ -6,6 +6,12 @@ from libcpp.string cimport string
 from cpython cimport PyObject, Py_XINCREF, Py_XDECREF, \
     PyObject_CallMethodObjArgs
 
+DEF HAVE_SAGE = ${HAVE_SAGE}
+have_sage = ${HAVE_SAGE}
```

Comment:
Yeah, syntax highlighting is a concern. `DEF HAVE_SAGE` is a compile time definition, like `#define` in C. `from sage.rings cimport integer as sageInteger` can only be done if sage is there at compile time. If it's not `try catch` won't catch it as `try catch` is not run at compile time.